src/server/pfs/server/driver.go

```diff
@@ -759,18 +759,10 @@ func (d *driver) writeFinishedCommit(ctx context.Context, commit *pfs.Commit, co
 		if err := repos.Get(commit.Repo.Name, repoInfo); err != nil {
 			return err
 		}
-		for _, branch := range repoInfo.Branches {
-			if branch.Name == "master" {
-				branchInfo := &pfs.BranchInfo{}
-				if err := d.branches(commit.Repo.Name).ReadWrite(stm).Get(branch.Name, branchInfo); err != nil {
-					return err
-				}
-				if branchInfo.Head.ID == commit.ID {
-					repoInfo.SizeBytes = commitInfo.SizeBytes
-					if err := repos.Put(commit.Repo.Name, repoInfo); err != nil {
-						return err
-					}
-				}
+		if commitInfo.Branch.Name == "master" {
```

Comment:
See outdated comment, but I'm not sure that this will work. In the deferred processing case for example, we'll never have any commits created on the `master` branch, everything gets created on `staging`, but we'll still assign a head to `master` with `create-branch` and thus still want to report a non-zero size for the repo.

Reply:
Yep, reverted it back to the original solution